ai/interview/company_job_posting.py

```
@@ -0,0 +1,187 @@
+        ("user", jd_text)
+    ])
+
+    settings = get_settings()
+    llm = ChatOpenAI(
+        model="gpt-4o",
+        temperature=0.1,
+        api_key=settings.OPENAI_API_KEY
+    )
+
+    # 단순 dict 반환
+    response = (prompt | llm).invoke({})
```

Ruff (0.13.3)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)